example.v0.tex

```diff
@@ -0,0 +1,95 @@
+\documentclass{refsheet}
```

Comment:
Please remove the associated PDF

Reply:
Probably remove all of the example, changing also the tex Q&A would be a good option.